server/DataStorageController.js

```diff
@@ -0,0 +1,169 @@
+/* DataStorageController.js
+
+   This module is used as an interfact to the database.
+   All database reads or updates should be done through this module.
+
+   Trivial Purfuit
+   Team Alpha
+*/
+
+// Default DB parameters
+require('dotenv-safe').config();
+const host = process.env.DB_HOST;
+const user = process.env.DB_USER;
+const password = process.env.DB_PASSWORD;
+const database = process.env.DB_DATABASE;
+
+// Establish a connection to the database
+const mySQL = require('mysql')
+const con = mySQL.createConnection({
```

Comment:
I think we want this connection to open up when we start up the server so we can make calls to the db, right?

Reply:
In Teams we commented about this. Unless you want to change it, leaving it like this is all gud